sdk-extensions/autoconfigure/src/test/java/io/opentelemetry/sdk/autoconfigure/OtlpConfigUtilTest.java

```diff
@@ -58,4 +62,39 @@ void getOtlpProtocolDefault() {
                         "otel.exporter.otlp.traces.protocol", "qux"))))
         .isEqualTo("qux");
   }
+
+  @Test
+  void configureOtlpExporterBuilder_HttpProtobufEndpoint() {
+    assertThat(configureHttpProtobufEndpoint(DATA_TYPE_TRACES, "http://localhost:4317"))
+        .isEqualTo("http://localhost:4317/v1/traces");
+    assertThat(configureHttpProtobufEndpoint(DATA_TYPE_TRACES, "http://localhost:4317/"))
+        .isEqualTo("http://localhost:4317/v1/traces");
```

Comment:
Can you add a test for `http://localhost:4317/collector/v1/traces` -> `http://localhost:4317/collector/v1/traces` and `http://localhost:4317/collector` -> `http://localhost:4317/collector/v1/traces` ?

Reply:
This is asking for more than the spec specifies. Configuring a per-signal endpoint should not append anything, so `http://localhost:4317/collector` should stay as just that. As worded in the spec, the "/v1/traces" suffix is merely convention for the collector. If I have a backend that does not follow this convention, I should be able to override the per-signal endpoint with the per-signal configuration, without having anything changed in the URL.